python/scope.py

```diff
@@ -3,6 +3,7 @@
 import sys
 
 import lark
+import regex as re
```

Comment:
Oh is it not just `import re`?

Reply:
The [docs](https://docs.python.org/3/library/re.html) suggest it might be `import re` and that we've imported a different library here

Reply:
Yeah, I changed it